service/src/test/java/bio/terra/drshub/models/DrsApiTest.java

```diff
@@ -46,4 +53,52 @@ void testSetHeader() {
 
     verify(apiClient).addDefaultHeader(name, value);
   }
+
+  @ParameterizedTest
+  @ValueSource(
+      ints = {
+        500, 502, 503, 504,
+      }) // HttpStatus.INTERNAL_SERVER_ERROR, BAD_GATEWAY, SERVICE_UNAVAILABLE, GATEWAY_TIMEOUT
```

Comment:
I think you can use java constants in this context
```suggestion
        HttpStatus.INTERNAL_SERVER_ERROR,
        HttpStatus.BAD_GATEWAY,
        HttpStatus.SERVICE_UNAVAILABLE,
        HttpStatus.GATEWAY_TIMEOUT
      })
```

Reply:
unfortunately no because `ValueSource` only supports primitives and those constants are not